### ISSUE TITLE
o-banner: low contrast on focus state of button - ID: DAC_Non_Text_Contrast_01

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/origami#503]
# Issue
This is from page 27 of the [audit report](https://drive.google.com/file/d/17wq6b0CIX41RA4muBFDzUDxE0-qzHBIH/view)

Issue can be found on: <https://www.ft.com/__origami/service/build/v3/demo?component=o-banner%404.0.0&demo=banner&system_code=origami-repo-data&brand=master>

When the ‘Try it now’ link receives keyboard focus, the focus indicator fails to meet WCAG AA guidelines for non-text contrast. This issue may affect low-vision users, who may find difficulty in distinguishing the grey focus indicator against the turquoise background colour of the toggle button. To meet AA guidelines for non-text contrast the minimum required contrast ratio is 3.1 (it is currently 2.1:1).

# Proposed Solution:
Ensure the border colour meets or exceeds the minimum required contrast ratio of 3.1 against adjacent colours in order to satisfy WCAG 2.1 Level AA criteria. This will enable low- vision and keyboard only users to be able to identify where the focus is on the page correctly.

**Comment by JakeChampion** [comment]
This issue comes from o-buttons and I think from this part of the code in particular --> https://github.com/Financial-Times/origami/blob/c1cb152c2e82c51428478628b2e0994da585f639/components/o-buttons/src/scss/_functions.scss#L182

**Comment by JakeChampion** [comment]
This issue is also present on <https://www.ft.com/__origami/service/build/v3/demo?component=o-banner%404.0.0&demo=small&system_code=origami-repo-data&brand=master> and found on page 29 of the audit report

**Comment by JakeChampion** [comment]
This issue is also present on <https://www.ft.com/__origami/service/build/v3/demo?component=o-banner%404.0.0&demo=product&system_code=origami-repo-data&brand=master> and found on page 34 of the audit report

**Comment by notlee** [comment]
related designs: https://www.figma.com/file/FLsNMzScj76auCGzEXHFDL/focus-ring?node-id=0%3A1

**Comment by notlee** [comment]
This is reflected in our UI Kit. We plan to update all focus outlines

https://www.figma.com/file/MyHQ1qdwYyek5IBdhEEaND/FT-UI-Library?node-id=0%3A915
![Screenshot 2022-08-24 at 16 38 59](https://user-images.githubusercontent.com/10405691/186461641-d8949a61-42c4-4992-a6c9-5cbc9697dc93.png)

**Comment by notlee** [comment]
Solved by this: https://github.com/Financial-Times/origami/issues/504

**Comment by notlee** [comment]
https://github.com/Financial-Times/origami/pull/842
